tagging/images_hierarchy.py

```diff
@@ -25,41 +39,37 @@ class ImageDescription:
         parent_image=None,
         taggers=[
             SHATagger,
-            UbuntuVersionTagger, PythonVersionTagger,
-            JupyterNotebookVersionTagger, JupyterLabVersionTagger, JupyterHubVersionTagger
```

Comment:
This was better, because there were nice groups

Reply:
It would also be more complexity to maintain as the groups you observe isn't obvious to everyone.

For example, why is UbuntuVersionTagger next to PythonVersionTagger but not SHATagger? I'm opinionated in favor of not trying to create and maintain structure here, and I have not yet learned about a way to force black to ignore formatting on a section but I would prefer not to try do so even if it was possible.

Reply:
I guess one could create groups by first creating lists and then adding lists together.